frameworks/OCaml/opium/src/lib/pool.ml

```diff
@@ -0,0 +1,28 @@
+let url = "localhost"
+let port = 5432
+let database = "opi"
+let connection_uri = Printf.sprintf "postgresql://%s:%i/%s" url port database
+
+let connect () =
+  connection_uri
+  |> Uri.of_string
+  |> Caqti_lwt.connect_pool ~max_size:10
+  |> function | Ok pool   -> pool
+              | Error err -> failwith (Caqti_error.show err)
```

Comment:
This looks like unused testing code or am I reading it wrong?

Reply:
yeah, I missed this, thanks